### PR TITLE
little helper to get the int value of a html like color

### PR DESCRIPTION
**Pull request by key2peace**

### CODE REVIEW

**Review by davidcole1340 — CHANGES_REQUESTED** [review]
Hmm, I'm not sure if this belongs in the `Discord` class, maybe move it to the `src/Discord/functions.php` file and set `$colorTable` to be a `const`?

File: src/Discord/Discord.php [hunk 1]
```diff
@@ -290,6 +290,51 @@ class Discord
      */
     protected $client;
 
+    /**
+     * The HTML Color Table.
+     *
+     * @array HTML Color Table.
+     */
+    public $colorTable = [
+        'indianred' => 'cd5c5c', 'lightcoral' => 'f08080', 'salmon' => 'fa8072', 'darksalmon' => 'e9967a',
```
Comment (kjc75) [inline src/Discord/Discord.php:299]: Might also consider storing the values as their hex encoded value and save the string->int conversion via ```hexdec()``` on each use.

You'd then have ```'indianred' => 0xcd5c5c, ...```, and in the ```getColor()``` function you'd simply return ```$this->colorTable[$colorName]``` for found keys.

Comment (key2peace) [reply src/Discord/Discord.php:299]: will do that now

Comment (key2peace) [reply src/Discord/Discord.php]: fd320c2 contains @kjc75's sugestion
69649cb contains @davidcole1340's change request

**Review by davidcole1340 — APPROVED** [review]

**Pull request merged**